pkg/cmd/root.go

```diff
@@ -5,6 +5,9 @@ import (
 	"github.com/Azure/azure-workload-identity/pkg/cmd/serviceaccount"
 	"github.com/Azure/azure-workload-identity/pkg/cmd/version"
 
+	// import auth plugins
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
```

Comment:
nit: remove extra line and put comment beside the import statement
```suggestion
	_ "k8s.io/client-go/plugin/pkg/client/auth" // import auth plugins. See https://github.com/Azure/azure-workload-identity/issues/362.
```